sourcecode/hub/resources/views/components/layout.blade.php

```diff
@@ -1,9 +1,152 @@
 <x-layout.base :nav="$nav ?? true">
-    <main class="container">
-        <header>
-            <h1 class="fs-2">{{ $title }}</h1>
-        </header>
+    <div class="container-fluid">
+        <div class="row">
+            <!-- Sidebar -->
+            <nav id="sidebar" class="col-md-3 col-lg-2 d-md-block bg-white sidebar">
+                <div class="position-sticky">
+                    <!-- Sidebar content -->
+                    <div class="accordion p-3 m-0 border-0 bd-example m-0 border-0" id="filterAccordion">
+                        <div class="accordion-item">
+                            <h2 class="accordion-header">
+                                <button id="language" class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapseOne" aria-expanded="true" aria-controls="collapseOne">
```

Comment:
Check if language should be in the filter sidebar, there is also a dropdown in the design